components/stepper/themes/spectrum-two.css

```diff
@@ -17,21 +17,17 @@
 		--spectrum-stepper-border-color-default: var(--spectrum-gray-500);
 		--spectrum-stepper-border-color-hover: var(--spectrum-gray-600);
 		--spectrum-stepper-border-color-focus: var(--spectrum-gray-800);
-		--spectrum-stepper-border-color-focus-hover: var(--spectrum-gray-800);
+		--spectrum-stepper-border-color-focus-hover: var(--spectrum-gray-900);
 		--spectrum-stepper-border-color-keyboard-focus: var(--spectrum-gray-900);
 
 		--spectrum-stepper-buttons-border-style: none;
 		--spectrum-stepper-buttons-border-width: 0;
-		--spectrum-stepper-buttons-border-color: var(--spectrum-gray-500);
-		--spectrum-stepper-buttons-background-color: var(--spectrum-gray-100);
-		--spectrum-stepper-buttons-border-color-hover: var(--spectrum-gray-600);
-		--spectrum-stepper-buttons-border-color-focus: var(--spectrum-gray-800);
-		--spectrum-stepper-buttons-border-color-keyboard-focus: var(--spectrum-gray-900);
 
 		--spectrum-stepper-button-border-width: var(--spectrum-border-width-100);
 
 		/** Invalid **/
 		--spectrum-stepper-border-color-invalid: var(--spectrum-negative-border-color-default);
+		--spectrum-stepper-border-color-hover-invalid: var(--spectrum-negative-border-color-hover);
```

Comment:
Looked like we were missing the invalid hover color ([token specs](https://www.figma.com/design/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?node-id=20523-6778))